src/main/java/org/example/camunda/process/solution/facade/FormsController.java

```diff
@@ -36,20 +33,22 @@ public class FormsController extends AbstractController {
   public JsonNode getFormSchema(
       @PathVariable String processDefinitionId, @PathVariable String formKey)
       throws TaskListException, IOException {
+
     if (formKey.startsWith("camunda-forms:bpmn:")) {
       String formId = formKey.substring(formKey.lastIndexOf(":") + 1);
       String schema = taskListService.getForm(processDefinitionId, formId);
       return JsonUtils.toJsonNode(schema);
     }
-    Form form = formService.findByName(formKey);
+
+    Form form = formService.findFormJsonFileByFormKey(formKey);
     return form.getSchema();
   }
 
   @GetMapping("/instanciation/{bpmnProcessId}")
   @ResponseBody
   public JsonNode getInstanciationFormSchema(@PathVariable String bpmnProcessId)
       throws IOException {
-    Form form = formService.findByName(bpmnProcessId);
+    Form form = formService.findFormJsonFileByFormKey(bpmnProcessId);
```

Comment:
Why is this change needed?

Reply:
Similar reasons as described above, I think the code related to loading forms should be updated so it's obvious whether a form is being loaded from json file, or from bpmn file, or from tasklist graphql. I think my latest force push changes this slightly.